docs/source/release/v5.1.0/mantidworkbench.rst

```diff
@@ -5,172 +5,270 @@ Workbench Changes
+- Use Jemalloc for memory allocation on Linux so memory can be released to the system.
 
-.. figure:: ../../images/instrument_view_sector.png
-   :align: right
-   :width: 400px
+**Launch**
+
+- There is a new profiling option in the workbench launch script, allowing for timing of start-up and other internal processes.
+- Improved loading of python plugins at start-up on slow disks.
+- User data directories are no longer checked at start-up, reducing launch times with slow network drives.
+
+**Scripting**
 
-- Added an option in the settings to specify the default legend size.
-- Added an option to the settings window to set the default colormap for image plots.
-- Colorfill plots of very large workspaces now take less than 10 seconds to plot
-- Improved loading of python plugins at startup on slow disks.
-- Added a circular sector shape in the Pick and Mask tab of the instrument view.
-- Workbench will now spot if it is about to create the settings window off the available screen, and will move it so it is all visible. This is important as it is a modal dialog and could freeze the application in an unrecoverable way before.
-- Sliceviewer no longer lists the reversed colourmaps along with the regular, instead they are accessed with a reverse checkbox.
-- Sliceviewer colourmap uses the default colourmap from the settings.
-- Sliceviewer scale remains set when reopened
 - Code completions are now loaded when the code editor is first changed.
-- The ExtractFFTSpectrum algorithm has extra properties (Shift, AutoShift, and AcceptXRoundingErrors) to reflect those of the underlying FFT.
-- Legends in 1D plots are now editable in-situ.
-- Added an option in `figure options` to change the canvas color for an axis.
-- Sliceviewer cut line plots' line widths reduced
-- When line plots are active on the sliceviewer the arrow keys can now be used to move the cursor a pixel at a time.
-- When showing monochromatic workspaces, the instrument widget will not show the integration bar, nor the pick widget the detector spectra graph.
-- In the instrument widget's rendering tab, added a Reset view button to restore to default projection.
-- In the instrument widget's draw tab, added the option to mask, draw ROI and group single pixel and tube.
-- `TableWorkspaces` can now have columns containing errors linked to corresponding columns containing values, using the `setLinkedYCol(errColumn, dataColumn)`.
-- Reduce log level of project recovery when it fails to save a plot correctly, to debug from warning level, to avoid excess messages in the console.
-- Added a Clear button to the workspaces toolbox which removes all workspaces from the tree including hidden ones
+- Variables assigned in python scripts are now cleared when a script is run in its entirety.
+- When a running script is aborted, Mantid will automatically attempt to cancel the current algorithm rather than wait for it to complete. IOn some cases, this can result a lot of time saved!
```

Comment:
```suggestion
- When a running script is aborted, Mantid will automatically attempt to cancel the current algorithm rather than wait for it to complete. On some cases, this can result a lot of time saved!
```